Applications/JupyterHub.qmd

```diff
@@ -81,7 +88,9 @@ You can install additional Python packages if necessary. This can be done both f
 
 If you prefer to work with a separate file, you can create a new Markdown (.md) or text (.txt) from the launcher or right click on the side bar and New file. Open the file and start writing your Markdown content using the Markdown syntax. Once you've written your content, you can save the file and preview it by double-clicking on it in the file browser, which will open a preview pane showing the rendered Markdown.
 
-Alternatively, you can include Markdown and rich text directly within Notebooks. In a notebook, you can create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing  `M`. Then, you can write your Markdown content in the cell using the Markdown syntax. To preview the rendered Markdown, you can either run the cell by pressing `Shift` + `Enter`, or use the "Preview" option from the cell's context menu to see the rendered content without executing the cell's code.
+Alternatively, you can include Markdown and rich text directly within Notebooks. 
+In a notebook, you can create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing  `M`. 
+Here, you can write your Markdown content in the cell using the Markdown syntax. Then execute the cell by pressing `Shift` + `Enter` .
```

Comment:
@soxofaan Yes, I did update the sentence as shown below, which is reflected in made changes but not sure why it is not showing in the document. I will redo it.

 Alternatively, you can include Markdown and rich text directly within Notebooks. 
In a notebook, you can create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing  `M`. 
Here, you can write your Markdown content in the cell using the Markdown syntax. Then execute the cell by pressing `Shift` + `Enter` .

Reply:
I updated the sentence as you suggested. 😅